crates/cvm-agent-models/src/lib.rs

```diff
@@ -46,6 +53,19 @@ pub mod bootstrap {
         /// The optional server.
         pub server: Option<String>,
     }
+
+    /// The heartbeat configuration.
+    #[serde_as]
+    #[derive(Clone, Deserialize, Serialize)]
+    pub struct HeartbeatConfig {
+        /// The interval at which heartbeats should occur.
+        #[serde_as(as = "DurationSeconds")]
+        pub interval: Duration,
+
+        /// The private key to use to pay for heartbeat transactions.
+        #[serde_as(as = "Hex")]
+        pub wallet_private_key: Vec<u8>,
+    }
```

Comment:
Does this originate from the builder telling the agent: "this is how frequently you should be sending HTXs and from this key"?

Reply:
For now this is part of the agent's config. Eventually it will likely be the user specifying this.